examples/pytorch/dagnn/utils.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+import random
+from torch.nn import functional as F
+import torch
+
+
+def evaluate(model, graph, feats, labels, idxs):
+    model.eval()
+    with torch.no_grad():
+        logits = model(graph, feats)
+        results = ()
+        for idx in idxs:
+            loss = F.cross_entropy(logits[idx], labels[idx])
+            acc = torch.sum(logits[idx].argmax(dim=1) == labels[idx]).item() / len(idx)
+            results += (loss, acc)
+    return results
+
+
+def generate_random_seeds(seed, nums):
+    random.seed(seed)
```

Comment:
`random.seed(seed)` is redundant.

Reply:
If remove this code, the generated seeds will change every time.

Reply:
Fine.